lib/bgsite_official/home/websites.ex

```diff
@@ -11,14 +11,15 @@ defmodule BgsiteOfficial.Home.Websites do
     field :likes, :integer
     field :title, :string
     field :urls, :string
+    field :tag, :string
```

Comment:
This is a nice idea, but you have to consider renaming the column name in the database table. It has to be done by a new database migration to rename the column.